doc/user/content/sql/alter-cluster.md

```diff
@@ -15,8 +15,22 @@ cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
 
 {{< diagram "alter-cluster-reset.svg" >}}
 
+### `with_options`
+
+{{< private-preview />}}
```

Comment:
QQ: w.r.t. the private preview features, can we highlight them in other pages?
For example, in https://materialize.com/docs/concepts/clusters/#sizing-your-clusters, we have 

> Resizing incurs downtime, as it requires all objects in the cluster to hydrate. In the future, it will be possible to resize clusters with no downtime ([#20010](https://github.com/MaterializeInc/materialize/issues/20010)).

Can we mention that this is available as private preview? or do we generally wait until things are out of private preview before mentioning them more broadly in the docs?

Reply:
I was just about to ask about this! I'm not sure if we should call this out here or wait until we public preview.

Reply:
@kay-kim, we for sure can. Just be mindful to always include some form of note about it being PrPr.

Reply:
👍 I'll add a patch that calls it out in our clusters concepts page.